test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -258,8 +281,10 @@ void ProtoConverter::visit(VarDecl const& _x)
 	m_output << "let " << varName << " := ";
 	visit(_x.expr());
 	m_output << "\n";
-	m_scopeVars.top().push_back(varName);
-	m_variables.push_back(varName);
+	if (m_inFunctionDef)
+		m_funcVars.back().back().push_back(varName);
```

Comment:
Is it sure that `m_funcVars` has size at least [1][1] here?

Reply:
Nevermind, just re-read the `open*Scope` functions

Reply:
Right, will fix.